pkg/build/apk.go

```diff
@@ -85,9 +89,31 @@ func (bc *Context) InitApkKeyring() (err error) {
 	for _, element := range keyFiles {
 		log.Printf("installing key %v", element)
 
-		data, err := os.ReadFile(element)
+		asURI := uri.New(element)
+		asURL, err := url.Parse(string(asURI))
```

Comment:
Why are you parsing it twice?

Reply:
The first time is to normalize the input into a URI, which can then be used with `net/url`.

Reply:
I added a comment explaining it.